nin/backend/nin

```diff
@@ -63,6 +64,29 @@ program
     generate.generate(projectRoot, resource, name);
   });
 
+program
+  .command('update')
+  .description('Update nin to newest version')
+  .action(function() {
+    echo('Checking for new version of nin');
+
+    // normalize nin path to point to /path/to/nin-project/nin
+    var ninExecutable = __dirname;
+    var ninPath = ninExecutable.split('/');
+    var ninPath = ninPath.splice(0, ninPath.length-2).join('/');
+
+    cd(ninPath);
```

Comment:
I did include it as global in the top of the file, which is why these functions are here.

I can change it to locally import them, which will add the prefix.